onnx/defs/math/defs.cc

```diff
@@ -890,24 +890,27 @@ ONNX_OPERATOR_SET_SCHEMA(
           matmulShapeInference(ctx, 0, 1);
         }));
 
-static const char* TopK_ver10_doc = R"DOC(
-Retrieve the top-K elements along a specified axis. Given an input tensor of
+static const char* TopK_ver11_doc = R"DOC(
+Retrieve the top-K largest or smallest elements along a specified axis. Given an input tensor of
 shape [a_1, a_2, ..., a_n, r] and integer argument k, return two outputs:
   -Value tensor of shape [a_1, a_2, ..., a_{axis-1}, k, a_{axis+1}, ... a_n]
     which contains the values of the top k elements along the specified axis
   -Index tensor of shape [a_1, a_2, ..., a_{axis-1}, k, a_{axis+1}, ... a_n] which
    contains the indices of the top k elements (original indices from the input
    tensor).
-   
-Given two equivalent values, this operator uses the indices along the axis  as
+
+If "mode" is 1 (the default value) then the k largest elements are returned.
+If "sorted" is 1 (the default value) then the resulting k elements will be sorted.
```

Comment:
I'd suggest to use `order` as the attribute name and that attribute is an enum of `"ascent"` and `"descent"`.

Reply:
I'd prefer changing the attribute-name to "largest" (or something like that). "Order/descent/ascent" is still slightly ambiguous whether it is talking about which elements are selected or about the order in which the selected elements appear in the output.

Reply:
It may also be useful to say that the attribute value must be 0 or 1 (to avoid dealing with ambiguous cases like a value like 2).

Reply:
@gramalingam, this operator can return the largest `K` or the smallest `K` elements depending on the attribute `mode`, so `largest` looks ambiguous.

Reply:
"largest" is a Boolean flag. So, if "largest=1" return K largest. If "largest=0" return K smallest. Or, use a string-valued "mode" with possible values as "largest" or "smallest" … but I think Boolean-flags are simpler than string-valued flags.

Reply:
Your `largest` attribute is the `mode` attribute in this PR. This PR's `sorted` attribute indicates if the returned values should be sorted.

Reply:
Okay, it looks like we were talking about two different attributes. I was suggesting renaming the "mode" attribute to "largest" (because mode=1 or 0 doesn't tell me what it means). It looks like you are talking about changing the "sorted" attribute. Sorry about the confusion.

Reply:
I changed attribute name back from 'mode' to 'largest'. 
Regarding attribute name 'sort', it is redundant to name it with ascending or descending because order is already implied by the 'largest' attribute. I updated description for sort to make it clearer:
'If "sorted" is 0, order of returned 'Values' and 'Indices' are undefined.'

Reply:
How about extending "resulting k elements will be sorted." to "resulting k elements will be sorted, in descending order if largest=1 and in ascending order if largest=0."?